test/EmptyApp.FunctionalTests/project.json

```diff
@@ -1,31 +1,30 @@
 ﻿{
   "webroot": "wwwroot",
-  "version": "1.0.0-*",
+  "version": "1.0.0-rc2",
```

Comment:
That makes sense, and yes we don't publish it anywhere. I will remove that. Thanks!